### PR TITLE
actualize code for updated version of winregistry

### PR DESCRIPTION
**Pull request by shpaker**
fix https://github.com/keygen-sh/py-machineid/issues/23

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
   author=__author__,
   author_email='oss@keygen.sh',
   license='MIT',
-  install_requires=['winregistry; sys_platform == "win32"'],
+  install_requires=['winregistry>=2.01; sys_platform == "win32"'],
```
Comment (ezekg) [inline setup.py:17]: Can we add a < v3 constraint here too? So we can avoid this issue again in the future.

Comment (shpaker) [reply setup.py]: Hi, of course I'll do it

File: reqs.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+winregistry>=2.0.1,<3.0.0
```
Comment (ezekg) [inline reqs.txt:1]: Did you mean to include this?

Comment (shpaker) [reply reqs.txt]: 😯

Comment (shpaker) [reply reqs.txt]: fixed

**Comment by ezekg** [comment]
Thanks for the PR! Published in v0.7.0.

**Pull request merged**